ui/client/datasets/DataTransformation/RunTransformations.js

```diff
@@ -40,7 +40,7 @@ const RunTransformations = ({
   setAllTransformationsComplete,
   setTransformationProcessingError,
 }) => {
-  const classes = useStyles();
+  const { classes } = useStyles();
```

Comment:
These styles weren't getting applied at all previously because of the incorrect import. Now they are and it looks better.